tests/GithubApp.py

```diff
@@ -98,9 +98,21 @@ def testGetPublicApp(self):
         self.assertEqual(app.updated_at, datetime(2019, 12, 10, 19, 4, 12))
         self.assertEqual(app.url, "/apps/github-actions")
 
+
+class GithubAppAuth(Framework.TestCase):
+    def setUp(self):
+        self.appAuthMode = True
```

Comment:
This sets auth mode for all subsequent tests, this should only be used by `tests.conftest.pytest_configure` via command line arguments like `--record`.

Please use the same way of authentication as used in `tests/GithubIntegration.py`:
```python
        auth = github.Auth.AppAuth(APP_ID, PRIVATE_KEY)
        g = github.Github(auth=auth)
        app = g.get_app()
```

You can import `APP_ID` and `PRIVATE_KEY` from `tests/GithubIntegration.py`.

So this `GithubAppAuth` class and `def setUp` can be removed.

Reply:
ok, I removed this code from the PR then as it is not required. This feature would then be orthogonal to this PR.